unit-tests/func/hw-errors/l500/l500-error-common.h

```diff
@@ -0,0 +1,95 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../../../test.h"
+#include <concurrency.h>
+
+using namespace rs2;
+
+const uint8_t l500_trigger_error_opcode = 0x5E;
+
+// Possible error values
```

Comment:
These have to match the `l500_notifications_types` in `l500-private.h`, right?
Please mention this

Reply:
(both here and there)

Reply:
Also: why do we redefine this map when there's a map in `l500-private.h`?

Reply:
I added also 10,11,13 that does not exist on l500-private.h map we give it general description